lib/fs.js

```diff
@@ -454,7 +454,20 @@ function openSync(path, flags, mode) {
   return result;
 }
 
-function read(fd, buffer, offset, length, position, callback) {
+// usage:
+// fs.read(fd, [buffer[, offset[, length[, position]]], callback);
+function read(fd, ...args) {
+  let callback = args.pop();
+  // Not really thrilled with this, but done to satisfy the linter
+  // Maybe adding a linter exception here might be better?
```

Comment:
what's this? what linter error?

Reply:
if i do something like this instead 
```
let [
    buffer = Buffer.alloc(16384),
    offset = 0,
    length = buffer.length,
    position = null ] = args;
```

The linter complains with `buffer is never reassigned, use const instead`

Reply:
const [buffer = Buffer.alloc(16384);  
let [, offset = 0, length = buffer.length, position = null ] = args;

Reply:
:man_facepalming:  :smile: